common/src/main/java/com/revenuecat/purchases/common/offlineentitlements/PurchasedProductsFetcher.kt

```diff
@@ -57,11 +57,13 @@ class PurchasedProductsFetcher(
         val purchaseAssociatedToProduct = allPurchases.first { it.skus[0] == productIdentifier }
         val isActive = activeProducts.contains(productIdentifier)
         val expirationDate = getExpirationDate(isActive, purchaseAssociatedToProduct)
+        val mapping = productEntitlementMapping?.mappings?.get(productIdentifier)
         return PurchasedProduct(
             productIdentifier,
+            mapping?.basePlanId,
```

Comment:
The idea is that this will be used to create the offline customer info, but that part is not done yet. @vegaro started this in a different PR #959